docs/src/integrations/exchange.md

```diff
@@ -488,10 +488,121 @@ You can also doublecheck whether a particular blockhash is still valid by sendin
 request with the blockhash as a parameter. If the response value is null, the
 blockhash is expired, and the withdrawal transaction should never succeed.
 
+## Supporting the SPL Token Standard
+
+[SPL Token](https://spl.solana.com/token) is the standard for wrapped/synthetic
+token creation and exchange on the Solana blockchain. 
+
+The SPL Token workflow is similar to that of native SOL tokens, but there are a
+few differences which will be discussed in this section.
+
+### Token Mints
+
+Each *type* of SPL Token is declared by creating a *mint* account.  This account
+stores metadata describing token features like the supply, number of decimals and
+various authorities with control over the mint.  Each SPL Token account references
+its associated mint and may only interact with SPL Tokens of that type.
+
+### Installing the `spl-token` CLI Tool
+
+`spl-token` is distributed from Rust [crates.io](https://crates.io) via the Rust
+`cargo` command line utility. The latest version of `cargo` can be installed
+using a handy one-liner for your platform at [rustup.rs](https://rustup.rs). Once
+`cargo` is installed, `spl-token` can be obtained with the following command:
+
+```
+cargo install spl-token-cli
+```
+
+You can then check the installed version to verify
+
+```
+spl-token --version
+```
+
+Which should result in something like
+
+```text
+spl-token-cli 2.0.1
+```
+
+### Account Creation
+
+SPL Token accounts impose additional requirements as opposed to native System
+Program accounts.
+
+1. SPL Token accounts are not implicitly created, so must be created explicitly
+before an SPL Token balance can be deposited
+1. SPL Token accounts must remain [rent-exempt](https://docs.solana.com/apps/rent#rent-exemption)
+for the duration of their existence and therefore require a small amount of 
+native SOL tokens be deposited at account creation. For SPL Token v2 accounts,
+this amount is 2,039,280 Lamports (0.00203928 SOL).
+
+When a user wishes to deposit a new SPL Token type, a new account will need to be
+created for that user at the given mint.
```

Comment:
I think we should just go all-in on the recommendation that the exchange pre-create deposit accounts in a large batch.    

This brings us to a new `solana-tokens` feature -- `distribute-tokens` (but real tokens, not lamports).  Useful here but it also unlocks SPL Token airdrop campaigns too.